cmd/relayproxy/config/config.go

```diff
@@ -91,10 +92,15 @@ func New(flagSet *pflag.FlagSet, log *zap.Logger, version string) (*Config, erro
 			log.Error("error loading file", zap.Error(errBindFile))
 		}
 	}
-
+	configMap := k.Raw()
```

Comment:
If we keep this outside of the function we are not able to pass multiple environment variables.

If we have something like
```
RETRIEVERS_1_KIND=github
RETRIEVERS_1_REPOSITORYSLUG=thomaspoignant/go-feature-flag
RETRIEVERS_1_BRANCH=main
RETRIEVERS_1_FILEPATH=testdata/flag-config.yaml
```

Only the last environment variable will be taken into consideration.

_(check proposal bellow to make it work)_

```suggestion
```